Marlin/stepper.cpp

```diff
@@ -459,6 +474,10 @@ void Stepper::isr() {
       // #if ENABLED(ADVANCE)
       //   e_steps[TOOL_E_INDEX] = 0;
       // #endif
+
+      #if ENABLED(MK2_MULTIPLEXER)
+        select_multiplexed_stepper(current_block->active_extruder);
```

Comment:
It seems prudent not to call this if the active extruder hasn't changed. Also, it borders on Very Bad Idea to include this function call (or inlined set of pin writes) in every single ISR invocation. This approach has already been specifically rejected for this reason. Having a multiplexer should add zero overhead here.